docs/application/resource.md

```diff
@@ -12,23 +12,27 @@ Resources belong to an environment, and you can view the list of resources in th
 2. By default, you will enter the view of the first environment. You can switch to view resources under different projects or environments through the top-level breadcrumb. You can also set the default environment to enter through the breadcrumb.
 3. In the `Resources` tab, you can view the list of resources under this environment.
 
-![resource-list](/img/v0.5.0/application/resource/resource-list.png)
+![resource-list](/img/v0.6.0/application/resource/resource-list.png)
 
 ## View Resource Details
 
 1. Navigate to the `Resources` tab in the environment view.
 2. Find the resource you need to view and click its name to enter the detail page.
 
+On the resource details page, you can view basic information of the resource, the list of components, dependency graphs, etc. You can also view the latest run records or edit the resource.
```

Comment:
On the resource details page, you can view the resource's basic information, component list, dependency graphs, and more.